rust/pjdfstest.toml

```diff
@@ -5,18 +5,20 @@
 # A list of these features is provided when executing the runner with `-l`.
 [features]
 # File flags can be specified for OS which supports them.
-# file_flags = ["UF_IMMUTABLE"]
-
+file_flags = ["SF_IMMUTABLE", "SF_NOUNLINK", "SF_APPEND"]
 # Here is an example with the `posix_fallocate` syscall.
-posix_fallocate = {}
 
 # Might use the key notation as well.
-[features.posix_fallocate]
+[features.chflags]
 
 [settings]
 # naptime is the duration of various short sleeps.  It should be greater than
 # the timestamp granularity of the file system under test.
 naptime = 0.001
+# Enable the EROFS tests, which temporarily remount the file system as read-only.
+# erofs = true
+# Path to a secondary file system for tests which require another than the mounted one (EXDEV for example).
+# secondary_fs = "/home/musikid/other_fs"
```

Comment:
Since we specify the first file system on the command line, can we specify the second in the same way?  Or instead, specify both in the config file?

Reply:
It's also possible to specify it on the command line.